tests/lib/config/config-ops.js

```diff
@@ -0,0 +1,407 @@
+/**
+ * @fileoverview Tests for ConfigOps
+ * @author Nicholas C. Zakas
+ * @copyright 2015 Nicholas C. Zakas. All rights reserved.
+ * See LICENSE file in root directory for full license.
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var assert = require("chai").assert,
+    assign = require("object-assign"),
+    environments = require("../../../conf/environments"),
+    ConfigOps = require("../../../lib/config/config-ops");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+describe("ConfigOps", function() {
+
```

Comment:
Sorry for nitpicking, but block padding in this file seems to be all over the place. Maybe we should enable `block-padding` rule on our repository?